### DISCUSSION TITLE
DDEV setup

### DISCUSSION THREAD

**Discussion by gways** [wintercms/winter#1278] [Q&A] [answered]
I have just started using ddev. Could someone provide a 'CMS Quickstart' for WinterCMS? Something like: https://ddev.readthedocs.io/en/stable/users/quickstart/#joomla? Thanks!

**Comment by mjauvin** [comment, marked as answer]
https://wintercms.com/docs/v1.2/docs/architecture/using-composer#installing-winter-via-composer

**Reply by gways** [reply]
Thanks! Not 100% the answer to my question, but still helpfull.

**Comment by LukeTowers** [comment]
@jaxwilko might be able to add some additional insight, from looking at [Laravel's instructions](https://ddev.readthedocs.io/en/stable/users/quickstart/#laravel) you might want to do some extra steps to make sure Vite is fully supported, and I'm not sure how well ddev handles the post create project composer commands that are included for simplifying initial project setup.

None of the core team uses ddev AFAIK though, so once you figure out a process that works for you if you would contribute it to their documentation that would be much appreciated!

Let me know if you have any questions!
